recipes/yappi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "yappi" %}
+{% set version = "0.98" %}
+{% set hash_type = "sha256" %}
+{% set hash = "5f657129e1b9b952379ffbc009357d0dcdb58c50f3bfe88ffbb992e4b27b263c" %}
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  entry_points:
+    - yappi = yappi:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Please add `toolchain` as a `build` requirement since this package has c-extensions.